experimental/templates/multiplatform-template/shared/build.gradle.kts

```diff
@@ -0,0 +1,75 @@
+plugins {
+    kotlin("multiplatform")
+    kotlin("native.cocoapods")
+    id("com.android.library")
+    id("org.jetbrains.compose")
+}
+
+version = "1.0-SNAPSHOT"
+
+kotlin {
+    android()
+
+    jvm("desktop")
+
+    ios()
+    iosSimulatorArm64()
+
+    cocoapods {
+        summary = "Some description for the Shared Module"
+        homepage = "Link to the Shared Module homepage"
+        ios.deploymentTarget = "14.1"
+        podfile = project.file("../iosApp/Podfile")
+        framework {
+            baseName = "shared"
+            isStatic = true
+        }
+        extraSpecAttributes["resources"] = "['src/commonMain/resources/**', 'src/iosMain/resources/**']"
+    }
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(compose.runtime)
+                implementation(compose.foundation)
+                implementation(compose.material)
+            }
+        }
+        val androidMain by getting {
+            dependencies {
+                implementation("androidx.appcompat:appcompat:1.5.1")
+                implementation("androidx.core:core-ktx:1.8.0")
+            }
+        }
+        val iosMain by getting
+        val iosTest by getting
```

Comment:
Let's remove it and `iosSimulatorArm64Test` until we support tests

Reply:
Removed